device/transport/http/src/http.ts

```diff
@@ -68,8 +68,8 @@ export class Http extends EventEmitter implements DeviceTransport {
   private _http: Base;
   private _opts: HttpReceiverOptions;
   private _cronObj: any;
-  private _intervalObj: number;
-  private _timeoutObj: number;
+  private _intervalObj: NodeJS.Timer;
```

Comment:
are these necessary for the Typescript upgrade? If not, I'd actually prefer them in their own PR.

Reply:
Yup, kinda are.  It started beefing about this.